pretyping/evarconv.ml

```diff
@@ -1228,6 +1228,7 @@ let solve_unconstrained_impossible_cases env evd =
   Evd.fold_undefined (fun evk ev_info evd' ->
     match ev_info.evar_source with
     | loc,Evar_kinds.ImpossibleCase ->
+       (* Why does this ad-hoc code have to be here instead of cases.ml? *)
```

Comment:
Eventually, a warning could be used, but I feel that the current compilation algorithm is too much heuristic-based and in-progress yet for a warning to be really meaningful. But yes, eventually, we could have a warning of the form "this case has been detected as contradictory; it can be omitted.".